zeppelin-test/src/test/java/org/apache/zeppelin/test/DownloadUtilsTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+
+//@Disabled("Takes a long time and depends on external factors.")
+class DownloadUtilsTest {
+
+  @Test
+  void downloadHadoop() {
+    String hadoopHome = DownloadUtils.downloadHadoop("3.4.0");
```

Comment:
seems it has not been tested yet. one of the major changes in 3.4 is switch AWS SDK from v1 to v2, that's kind of a breaking change.

Reply:
These are only test methods. It is up to the caller to decide which version is finally downloaded.